.travis.yml

```diff
@@ -0,0 +1,18 @@
+os: osx
+language: kotlin
+osx_image: xcode10.1
```

Comment:
does not need to be osx for kotlin. Check:
 - https://github.com/android10/Android-CleanArchitecture-Kotlin/blob/master/.travis.yml
 - https://docs.travis-ci.com/user/languages/java/
 - https://github.com/square/leakcanary/blob/master/.travis.yml

or other kotlin projects.

Reply:
A couple of things.  I am not assuming this project is android only.  I can certainly change the os, but I am not sure what the point is.